### PR TITLE
Creation of Lumi direct photon calorimeter in far backward system

### PR DESCRIPTION
**Pull request by aniakowalewska**
### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [x] New feature (issue #360 __)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [x] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
No
### Does this PR change default behavior?
No

### CODE REVIEW

File: compact/far_backward/lumi/directPC.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!-- SPDX-License-Identifier: LGPL-3.0-or-later -->
+<!-- Copyright (C) 2022 Anna B. Kowalewska -->
```
Comment (wdconinc) [inline compact/far_backward/lumi/directPC.xml:2]:
```suggestion
<!-- Copyright (C) 2023 Anna B. Kowalewska -->
```

File: src/LumiDirectPC_geo.cpp [hunk 2]
```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2022 Anna Kowalewska
```
Comment (wdconinc) [inline src/LumiDirectPC_geo.cpp:2]:
```suggestion
// Copyright (C) 2023 Anna Kowalewska
```

File: src/LumiDirectPC_geo.cpp [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  double        posX            = x_pos.x();
+  double        posY            = x_pos.y();
+  double        posZ            = x_pos.z();
+  double        rotX            = x_rot.x();
+  double        rotY            = x_rot.y();
+  double        rotZ            = x_rot.z();
+
+  Box box( sizeX, sizeY, sizeZ );
+  Volume vol( det_name + "_vol", box, desc.material( mat_name ) );
+  //vol.setVisAttributes( x_det.visStr() );
```
Comment (wdconinc) [inline src/LumiDirectPC_geo.cpp:41]:
```suggestion
```

File: src/LumiDirectPC_geo.cpp [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  vol.setVisAttributes(desc.visAttributes(x_det.visStr()));
+  vol.setSensitiveDetector(sens);
+
+  Transform3D  pos( RotationZYX(rotX, rotY, rotZ), Position(posX, posY, posZ) );
+
+  Volume motherVol = desc.pickMotherVolume( det );
+  PlacedVolume phv = motherVol.placeVolume( vol, pos );
+  phv.addPhysVolID("system", det_ID);
```
Comment (wdconinc) [inline src/LumiDirectPC_geo.cpp:49]: Where do you set `sector` and `module`? Do you not need them (is this detector not segmented; is it simply a single block of PbWO4)? If not, then remove from the readout pattern in the xml file.

**Review by wdconinc — CHANGES_REQUESTED** [review]
Looks good. Few changes.

File: src/LumiDirectPC_geo.cpp (hunk 4 above)
Comment (aniakowalewska) [reply src/LumiDirectPC_geo.cpp:48]: Yes, it is simply a single block of PbW04 detector. I have just removed sector and module part from the readout pattern in the xml file.

Comment (wdconinc) [reply src/LumiDirectPC_geo.cpp:48]: Thanks.

**Review by wdconinc — APPROVED** [review]

**Pull request merged**